packages/solana/src/deposit.ts

```diff
@@ -25,6 +34,17 @@ const getSolanaConnection = (solanaEndpoint: string) => {
     httpHeaders: headers,
     // for injecting responses in the tests
     fetch,
+    // fetch: async (...args) => {
```

Comment:
think we can revert

Reply:
I wanted to leave it because this is how you intercept the HTTP requests for the RPC node. I have to constantly add and remove it when adding new features